CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
-- Fix: modified JEXL transformations (toisostring, gettime, parseint, etc.) to return null instead of NaN when some unexpected situation occurs (#1701)
\ No newline at end of file
+- Add: index for Device model based on {service: 1, subservice: 1, id: 1, apikey: 1} (#1576)
```

Comment:
Do we have index documented in some .md file (either in this repo or IOTA-JSON)? In that case, documentation should be changed.

Reply:
Current doc is still valid: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#uniqueness-of-groups-and-devices